src/main/java/com/epam/architecture/roles/AdminRole.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.architecture.roles;
+
+import com.epam.architecture.userinterface.LibraryService;
+import jakarta.xml.soap.SOAPBody;
+import jakarta.xml.soap.SOAPException;
+import jakarta.xml.soap.SOAPFault;
+import jakarta.xml.ws.handler.soap.SOAPMessageContext;
+import jakarta.xml.ws.soap.SOAPFaultException;
+
+public class AdminRole implements LibraryRole {
```

Comment:
1. Rename
2. Make all fun static 
```suggestion
public class AuthorizationUtil {
```